tests/D2L.CodeStyle.Analyzers.Test/Specs/DependencyRegistrationsAnalyzer.cs

```diff
@@ -200,22 +219,15 @@ public struct RegistrationCallsInThisStructAreIgnored {
 			}
 		}
 
-		// Registrations in extension methods with enough information should be handled. 
-		public static class SomeExtensionMethods {

```

Comment:
Why is this deleted?  It looks like a requirement that is not duplicated elsewhere.  Can this come back?

Reply:
Discussed offline, we'll put in an extension method test in a separate PR.

Reply:
From an offline conversation, it sounds like this was broken already, so it wasn't testing what it claimed to test.  Removing for now, as it might take a lot of work to figure out how to fix it (especially if it may never have worked at all)